### PR TITLE
chore: add lpdaac_gedi_access tutorial

### PR DESCRIPTION
**Pull request by jjfrench**
- demonstrate how to convert lpdaac gedi https links to s3 links

### CODE REVIEW

File: docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb [hunk 1]
```diff
@@ -0,0 +1,258 @@
+{
```
Comment (wildintellect) [inline docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb:1]: <ol><li>Jamison, your name is missing from the Author list.</li><li>We need a big banner at the top about this being a temporary work around, as the S3 version isn't officially released yet.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (wildintellect) [inline docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb:1]: <p>Should we be doing a download here? Or just show loading or reading the data directly from S3 with h5py and/or xarray?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (jjfrench) [reply docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb:1]: Could switch it up, I was just keeping it consistent with the other "accessing" tutorials

Comment (wildintellect) [reply docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb:1]: But this isn't like other tutorials, avoiding downloading is the point of direct s3 access.

Comment (smk0033) [inline docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb:1]: <p>Neither of these links seem to work (... /docs/source/technical_tutorials/docs/source/technical_tutorials/search/granules.ipynb is what the links are doing). Might could also directly link the pages in the Docs site</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (wildintellect) [reply docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb:1]: relative path is probably `../search/notebookname.ipynb`

Comment (jjfrench) [reply docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb:1]: oopsies, meant to update these

**Comment by wildintellect** [comment]
Hmm, did we want to show the assume role in this one, or the temporary credential method  or both?

**Review by wildintellect — APPROVED** [review]

**Pull request merged**